src/graphql/shop/payment/paypal_standard.graphql

```diff
@@ -47,8 +47,7 @@ type PaypalFormFieldResponse {
     firstName: String! @rename(attribute: "first_name")
     lastName: String! @rename(attribute: "last_name")
     email: String!
-    address1: String!
-    city: String!
```

Comment:
Why remove this city,  check and if needed then fix it

Reply:
Added